index.bs

```diff
@@ -2224,6 +2231,10 @@ The following {{DOMException}} exceptions can be raised:
         the [=client=] does not support [[#sctn-related-origins|related origin requests]]
         or the [$related origins validation procedure$] failed.
 
+    :   {{SyntaxError}}
+    ::  <code>{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/language}}</code>
+        was present and was not a [=well-formed language tag=].
+
```

Comment:
If we agree to keep things consistent with `TypeError` as I call out above, we'll want to take this out.

Reply:
`SyntaxError` will stick around for now